cli/src/command/generate_config.rs

```diff
@@ -95,7 +95,7 @@ impl GenerateConfig {
     }
 
     /// If no port is provided by user, uses `DEFAULT_EXONUM_LISTEN_PORT`.
-    fn parse_external_address(input: &str) -> Result<SocketAddr, Error> {
+    pub fn parse_external_address(input: &str) -> Result<SocketAddr, Error> {
```

Comment:
Why `pub` ?

Reply:
I used this method in my own project for parsing command. This modification mistaken appears in this branch.